numpy/_core/tests/test_simd.py

```diff
@@ -185,7 +185,8 @@ def test_operators_logical(self):
         assert data_xnor == vxnor
 
     def test_tobits(self):
-        data2bits = lambda data: sum([int(x != 0) << i for i, x in enumerate(data, 0)])
+        def data2bits(data):
+            return sum([int(x != 0) << i for i, x in enumerate(data, 0)])
```

Comment:
I'm a bit surprised a blank line wasn't added here.

Reply:
These are automated fixes by ruff, and pycodestyle doesn't have anything to say about it at this point. I was thinking of handling this error in an upcoming PR that would handle the relevant `E` pycodestyle rule.